.github/workflows/previews.yaml

```diff
@@ -69,8 +69,8 @@ jobs:
           #   * `tr -s -`: squash repeated consecutive dashes into one.
           #   * `cut -b -39`: truncate the name to 39 characters, leaving 24 for
           #     "devbox-docs-preview-" and "-ref" for a max of 63.
-          safe_github_ref="$(echo "${GITHUB_REF_NAME}" | sed "s/[^0-9a-zA-Z]/-/g" | cut -b -39)"
-          namespace="$(echo "devbox-docs-preview-${safe_github_ref}-ref" | tr -s -)"
-          jetpack up docs \
+          safe_github_ref="$(echo "${GITHUB_REF_NAME}" | sed "s/[^0-9a-zA-Z]/-/g" | cut -b -32)"
+          namespace="$(echo "jetpack-io-preview-devbox-${safe_github_ref}-ref" | tr -s -)"
```

Comment:
Could we just do `devbox-docs-${safe_github_ref}-ref` (assuming these are the docs for devbox)

Or do we have a naming convention of `<org-name>-<environment>-<project>`?

Reply:
Must be prefixed with org name

Reply:
Alright, go for it.

Reply:
@loreto once we move to a private cluster model, we can do that? This is deploying to `jetpack-cloud`